src/mfakto.cpp

```diff
@@ -327,17 +327,20 @@ int init_CL(int num_streams, cl_int *devnumber)
     else for(i=0; i < numplatforms; i++) // autoselect: search for AMD
     {
       char buf[128] = {0};
+      cl_uint device_count;
+      int is_amd;
```

Comment:
I would normally write `bool` here, but I don't see a single `bool` in the code.